internal/cloud/testing.go

```diff
@@ -443,3 +444,54 @@ func testVariables(s terraform.ValueSourceType, vs ...string) map[string]backend
 	}
 	return vars
 }
+
+func TestCloudLocks(t *testing.T, a, b statemgr.Full) {
```

Comment:
This should be moved to state_test.go and probably not called as a helper but exist as a top level test.

Reply:
Not sure this test is called at all right now